examples/advanced/quantum_volume_errors.ipynb

```diff
@@ -1,134 +1,156 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        compiler=compiler,
</pre><p>Think you can just make this <code>cirq_google.optimized_for_xmon</code>now andremove compiler lambda?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>